src/eterna/folding/__tests__/Folder.test.ts

```diff
@@ -42,6 +43,105 @@ for (let folderType of [Vienna, Vienna2, NuPACK, LinearFoldV]) {
             .resolves.toBeUndefined();
     });
 
+    test('Nupack_Subopt_structures_noOligos', () => {    
```

Comment:
Couple issues here (applicable for both tests):
1) You're running this multiple times when we iterate through each folder type, even though it's always run against the same engine! Move it out of the loop so that we don't expand our test run times longer than necessary :)
2) Please follow the existing naming conventions for test names. This for example should be something like `NuPACK:suboptStructuresNoOligos`